dev/TabView/TabView.cpp

```diff
@@ -136,18 +139,21 @@ void TabView::OnApplyTemplate()
 
     if (SharedHelpers::IsThemeShadowAvailable())
     {
-        if (auto shadowCaster = GetTemplateChildT<winrt::Grid>(L"ShadowCaster", controlProtected))
+        if (!SharedHelpers::Is21H1OrHigher())
         {
-            winrt::ThemeShadow shadow;
-            shadow.Receivers().Append(GetShadowReceiver());
+            if (auto shadowCaster = GetTemplateChildT<winrt::Grid>(L"ShadowCaster", controlProtected))
+            {
```

Comment:
You're trying to get ShadowCaster, but shadow caster was removed down below:)

Reply:
Ah sorry @leonMSFT this was my mistake saying it was ok to remove the `ShadowCaster` and `ShadowReceiver` element in `TabView.xaml`. We will need to keep these in the scenario that an app is using the latest style on OS 21H1 or lower, where it will simply default to projected shadows.

Reply:
oh np, ok in that case I'll just leave caster/receiver in and let the 21H1 flag do the work